### PR TITLE
[MIG] mass_mailing_event: Migration to 10.0

### PR DESCRIPTION
**Pull request by chienandalu**
cc @Tecnativa

### CODE REVIEW

**Comment by yajo** [comment]
Please check Travis' ❌

**Comment by chienandalu** [comment]
It's raising `mail_tracking` and `mail_tracking_mailgun` errors. Not sure if they're related to this PR

**Review by pedrobaeza — APPROVED** [review]
Travis error unrelated

**Comment by chienandalu** [comment]
@Yajo Can you review again?

**Comment by pedrobaeza** [comment]
@rafaelbn please review

File: mass_mailing_event/models/mail_mass_mailing.py [hunk 1]
```diff
@@ -44,11 +44,10 @@ def _default_exclude_event_state_ids(self):
         string="Exclude", default=_default_exclude_event_state_ids)
 
     @api.model
```
Comment (yajo) [inline mass_mailing_event/models/mail_mass_mailing.py]: Are you sure this isn't `@api.multi` now?

**Review by yajo — CHANGES_REQUESTED** [review]
All code reviewed and it seems OK, just one little thing...

Besides, to get 100% coverage you need almost nothing more, maybe you want to go for that 😊

**Comment by chienandalu** [comment]
@Yajo Change done

**Review by yajo — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please go for full coverage with these steps (check https://codecov.io/gh/OCA/social/compare/69c40f846317761a490e93c85685c5919573811e...1888d3b58120c097d97f1a82107a8eb57173a43f/diff for diff info):

* Changing immediate `return` in favor of storing in the intermediate variable the result that is later returned.
* Testing the partner record count.

**Comment by chienandalu** [comment]
@pedrobaeza It's weird now `mass_mailing_custom_unsubscribe` tests are failing but only for Odoo (not for OCB)

**Comment by pedrobaeza** [comment]
The error is in linter:
```
mass_mailing_event/models/res_partner.py:21:1: W391 blank line at end of file
```

**Comment by chienandalu** [comment]
@pedrobaeza I swear yesterday was failing as I described :exclamation::confused::question:

**Comment by yajo** [comment]
@chienandalu you're not crazy: https://travis-ci.org/OCA/social/jobs/270497561

But if it goes 💚, just be happy.

**Comment by chienandalu** [comment]
:smile: Ok, all green now @Yajo @pedrobaeza

**Pull request merged**